docs/reduction-in-locks.md

```diff
@@ -0,0 +1,100 @@
+# Reduced backup lock time
+
+!!! important
+
+    The `--lock-ddl=REDUCED` option is a [tech preview](./glossary.md#tech-preview). Before using this option in production, we recommend that you test restoring production from physical backups in your environment, and also use the alternative backup method for redundancy.
+
+Percona Server for MySQL implemented `LOCK TABLES FOR BACKUP` to block Data Definition Language (DDL) statements on a server. Percona XtraBackup utilizes this lock during the backup process to ensure that any `DDL` event does not corrupt the backup. The lock does not affect the Data Manipulation Language (DML) statements.
+
+## How the backup lock works
```

Comment:
@patrickbirch @alina-derkach-oaza do you think we should remove this entire section? It should be placed in how xtrabackup works section. :thinking:

Reply:
Good point. I can create a separate doc under How Percona XtraBackup works and name it "How the backup lock works". In the Reduction in lock feature I can provide the link to that doc.

Reply:
For now, since this is a new feature and the end user needs to understand what has been changed, I'd leave this paragraph in the feature description.

Reply:
`Percona Server for MySQL implemented LOCK TABLES FOR BACKUP to block Data Definition Language (DDL) statements on a server. Percona XtraBackup utilizes this lock during the backup process to ensure that any DDL event does not corrupt the backup. The lock does not affect the Data Manipulation Language (DML) statements.`

Here we are talking about a backup lock and not giving the introduction about the feature. This elaborate statement is not strictly necessary. We could say something.

Percona Xtrabackup uses a backup lock (link to backup lock or some description) later to ensure that any DDLs on the server doesn't the corrupt the backup. This means, DDLs have to wait for the backup process to finish.

With reduced lock feature, xtrabackup would take lock for very short duration and the Server remains open to DDLs while the backup is in progress.

We should add that this feature also works with incremental backups.

@patrickbirch can you help us to rephrase?

Reply:
Benefits should come here

Reply:
I'll rewrite it, thank you

Reply:
done